docs/source/use_cases/annotate_variants/tute_annotation.rst

```diff
@@ -9,7 +9,8 @@ Annotate Variants with Tute Genomics
 To make use of this upon your own data:
 
 (1) First, load your data into Google Genomics and export your variants to BigQuery.  See :doc:`../load_data/index` for more detail as to how to do this.
-(2) Use the BigQuery `JOIN <https://cloud.google.com/bigquery/query-reference#joins>`_ command to join the Tute table with your variants and materialize the result to a new table.  Notice in the screenshot below the destination table and 'Allow Large Results' is unchecked.
+(2) Copy and modify one of the queries in See `Tute's documentation`_ so that it will perform a `JOIN <https://cloud.google.com/bigquery/query-reference#joins>`_ command against your table.
```

Comment:
Something is off with this sentence "Copy and modify one of the queries in See Tute’s documentation "